app/controllers/commitments_controller.rb

```diff
@@ -9,7 +9,7 @@ class CommitmentsController < ApplicationController
   skip_before_action :authenticate_user!, only: %i[index list show]
```

Comment:
Do we need to revert any of this? If not/it's now redundant, shall we remove the commented code?

Reply:
removed